### DISCUSSION TITLE
The control connection and stream conection should be initiated on the client side

### DISCUSSION THREAD

**Discussion by Lyon-Jiang** [alvr-org/ALVR#1496] [General]
The first downlink packet is not allowed In a typical network，unless the server and client are in the same LAN.
@zarik5

**Comment by Lyon-Jiang** [comment]
and i am streaming VR in a 5G private network，the control connection and stream conection initiated on the server side are blocked by firewall, so i have to config some iptables rules to allow them.

**Comment by zmerp** [comment]
To allow client/server discovery across LANs we need a discovery server. But ALVR currently does not have that, and so there is not even support in the code.
